config/initializers/devise.rb

```diff
@@ -5,6 +5,10 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class with default "from" parameter.
   config.mailer_sender = "pwreset@asudisc.org"
+  
+  # config secret key
+  config.secret_key = '46a8a008507d8c995c0487c0b2a2212dbb4244270e19df073941511456b84e
```

Comment:
Haven't done Rails in a while so not super familiar with this. 
Did you update devise and then it forced you to add this secret key? If so, we'd want to make this an environment variable so we're not committing our secrets. I added a new secret key to Heroku called DEVISE_SECRET so you should be able to just replace this key with ENV["DEVISE_SECRET"], then set an environment variable on your local (whatever you want)